index.html

```diff
@@ -242,7 +242,45 @@ <h1 class="headline-h1">About Me</h1>
         </ul>
       </li>
     </ul>
-  </div>                              
+  </div>
+  <div class="ContactPage">
+   <form action="https://formspree.io/f" method="post"></form>
+    <div class="contact-title">I'm always interested in hearing about new projects,
+    so if you'd like to chat please get in touch.</div>
+  <input class="namebox" type="text" id="name" name="user_name" placeholder="  Full name" maxlength="30" required>
+  <input class="namebox" type="email" id="email" name="user_email" placeholder="  Email adress" required>
+  <textarea name="comment" id="textA" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
```

Comment:
Your form validation is not working as expected because the `inputs` fields are not wrapped in `form`. Please ensure that all inputs are wrapped in the `form` element. And please ensure that validation is working as expected.